### PR TITLE
feat(icon-export): 12326 - Move figma-export config 

### PR DESCRIPTION
**Pull request by propakov**
https://kontur.fibery.io/Tasks/Task/Move-figma-export-script-to-ui-kit-package-from-root-12326

### CODE REVIEW

File: packages/default-icons/README.md [hunk 1]
```diff
@@ -45,3 +45,15 @@ You can choose the way you prefer:
    * SVGR can be run from the [CLI](https://react-svgr.com/docs/cli/)
    * SVGR can be used as a [webpack loader](https://react-svgr.com/docs/webpack/), this way you can import your SVG directly as a React Component
 
+### Export icons from Figma
+Before export icons, you will need to generate a Personal Access Token in Figma.
+
+1. Go to the File Browser.
```
Comment (IlyaIzr) [inline packages/default-icons/README.md:51]: File Browser? Like web browser?

**Review by IlyaIzr — APPROVED** [review]

File: packages/default-icons/package.json [hunk 2]
```diff
@@ -12,10 +12,18 @@
   "files": [
     "tslib"
   ],
-  "scripts": {},
+  "scripts": {
+    "figma:export": "figma-export use-config .figmaexportrc.cjs && prettier --write src/figma-icons/*",
+    "figma:export-env": "export FIGMA_TOKEN=$(grep '^FIGMA_TOKEN' .env.local | cut -d '=' -f2) && rm -rf src/figma-icons && figma-export use-config .figmaexportrc.cjs && prettier --write src/figma-icons/*"
```
[the remark this answers is not in the document]
Comment (propakov) [reply packages/default-icons/package.json:17]: Andrey added it, it just uses env file, but do the same.
Yeah, I'll add some description.

File: packages/default-icons/README.md (hunk 1 above)
Comment (propakov) [reply packages/default-icons/README.md:51]: files in figma

Screenshot 2022-09-20 at 14 17 27

Comment (propakov) [reply packages/default-icons/README.md:51]: ok, need to change it as it's unclear

Comment (propakov) [reply packages/default-icons/README.md]: updated with the link

**Pull request merged**